### PR TITLE
Updated docstring

### LINKED ISSUE #10635: Einops Reduce
Reformat general #791

### PR DESCRIPTION
**Pull request by satakshigarg**
closes #10635

### CODE REVIEW

**Comment by satakshigarg** [comment]
Let me know if there are any suggestions or changes required ?

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -1845,20 +1846,20 @@ def einops_reduce(
 
     Examples
     --------
-    With :class:`ivy.Array` input:
```
Comment (juliagsy) [inline ivy/functional/ivy/general.py:1848]: the original `input` should be fine here :)

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there! Thanks for the hard work! Just one small detail, otherwise looks good to me! Please request my review once again when you're done! Thanks a lot!

PS: One side note, just in case you haven't, please also check the equivalent array and container instance methods and ensure that the docstrings and docstring examples are correct! Great work if you had gone through them! Thank you!

File: ivy/functional/ivy/general.py (hunk 1 above)
Comment (satakshigarg) [reply ivy/functional/ivy/general.py]: Resolved it @juliagsy

**Review by juliagsy — APPROVED** [review]
thanks for contributing!

**Pull request merged**